packages/strapi-admin/admin/src/containers/AdminPage/index.js

```diff
@@ -71,11 +72,15 @@ export class AdminPage extends React.Component { // eslint-disable-line react/pr
 
   componentDidMount() {
     this.checkLogin(this.props);
+
+    ReactGA.initialize('UA-54313258-9');
```

Comment:
Will it be configurable? Or you will track all the Strapi projects?

Reply:
Sure thing. What if i dont want to track my project?

Reply:
Yap we plan to add a configuration to disable it of course ;)

Reply:
Done here https://github.com/strapi/strapi/pull/834